api/services/auth.js

```diff
@@ -1,9 +1,14 @@
 const nodemailer = require('nodemailer');
 const { hashHelpers, jwtHelpers, emailHelpers } = require('../helpers');
 const { ErrorHandler } = require('../middlewares/errorHandler');
+const ResetPasswordApproveError = require('../services/errorHandlers/resetPasswordApproveError');
+const ResetPasswordError = require('../services/errorHandlers/resetPasswordError');
+
 const errors = require('./errorHandlers/index');
 const db = require('../../database');
 
+const HOST = process.env.CL_HOST || 'http://localhost:3000';
```

Comment:
Why you make this variable global? I think it is not necessary to make this condition.

Reply:
I don't want to left the address of the server in code. It's not global it's a variable of environment